service/src/main/java/org/databiosphere/workspacedataservice/dao/PostgresJobDao.java

```
@@ -173,6 +174,19 @@ public GenericJobServerModel fail(UUID jobId, String errorMessage, Exception e)
     return update(jobId, StatusEnum.ERROR, errorMessage, e.getStackTrace());
   }
 
+  /**
+   * Mark a job as error without logging at the error level
+   *
+   * @param jobId id of the job to update
+   * @param errorMessage a short error message, if the job is in error
+   * @return the updated job
+   */
+  @Override
+  public GenericJobServerModel markError(UUID jobId, String errorMessage) {
```

I created a new method that does the same as `fail` except it uses `info` instead of `error` log level to avoid triggering Sentry; potentially I could instead add a `logError` flag to the `fail` method if someone has strong feelings about it.

---

I'm really waffling on this. I think it could actually be good after all to get alerts when jobs time out like this. If we accept that we're gonna get alerts, then you could do away with the `markError` method, which makes it even more appealing.

---

I suppose we could always go back to not alerting if we start getting the alerts and they're not useful.